### PR TITLE
CON-3056 Add subject for ``Quest01/listed``

### PR DESCRIPTION
**Pull request by Oumaimafisaoui**
[CON-3056](https://01talent.atlassian.net/browse/CON-3056)

[CON-3056]: https://01talent.atlassian.net/browse/CON-3056?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: subjects/DPxAI/listed/README.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+];
+
+// or for brevity, we often write them on a single line like so:
+
+let batteryLevels = [80, 60, 90, 50];
+```
+
+### Indexes in Arrays
+
+The position of an element in an array is called its index, starting from 0. So, our batteryLevels array is roughly equivalent to writing this object:
```
Comment (nprimo) [inline subjects/DPxAI/listed/README.md:45]: For consistency, `batteryLevels` should be formatted as the other variables in the subject

File: subjects/DPxAI/listed/README.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+You must declare a variable `components` that contains 4 strings, one for each robot component.
+
+#### Task 2:
+
+We provide you a variable `robotParts` that contains some elements. You will have to access them and assign their values to variables:
+
+- A variable `firstPart` for the first element of the robotParts list
+- A variable `lastPart` for the last element of the robotParts list
+- A variable `comboParts` as an array of 2 elements, the last and the first element of the robotParts list, in that order.
+  Example: if robotParts is `[1, 2, 3]`
```
Comment (nprimo) [inline subjects/DPxAI/listed/README.md:121]: `robotParts` should be formatted as the other variables in the subject for consistency

**Review by nprimo — CHANGES_REQUESTED** [review]
I left just a few format suggestions to keep the formatting in the subject consistent

File: subjects/DPxAI/listed/README.md (hunk 1 above)
Comment (Oumaimafisaoui) [reply subjects/DPxAI/listed/README.md]: Changed.

File: subjects/DPxAI/listed/README.md (hunk 2 above)
Comment (Oumaimafisaoui) [reply subjects/DPxAI/listed/README.md]: Changed.

**Review by nprimo — APPROVED** [review]
Once the last `roboParts` get formatted, you can resolve the conversation and merge :)

**Review by lotoussa — APPROVED** [review]

**Pull request merged**